arch/risc-v/src/mpfs/mpfs_shead.S

```diff
@@ -0,0 +1,87 @@
+/****************************************************************************
+ * /nuttx-bare/arch/risc-v/src/mpfs/mpfs_shead.S
```

Comment:
```suggestion
 * arch/risc-v/src/mpfs/mpfs_shead.S
```

Reply:
Wish my editor did not append the root folder name automatically there...

Reply:
Fixed